conans/client/generators/virtualenv.py

```diff
@@ -1,9 +1,106 @@
 import os
+import platform
+import textwrap
+
+from jinja2 import Template
 
 from conans.client.tools.oss import OSInfo
 from conans.model import Generator
 
 
+sh_activate_tpl = Template(textwrap.dedent("""
+    #!/usr/bin/env bash
```

Comment:
Noticed this...
I don't know why, but our GitLab executes its commands using POSIX shell, not bash.